sql/core/src/test/scala/org/apache/spark/sql/streaming/continuous/ContinuousSuite.scala

```diff
@@ -66,157 +66,115 @@ class ContinuousSuite extends ContinuousSuiteBase {
     val input = ContinuousMemoryStream[Int]
 
     testStream(input.toDF())(
-      AddData(input, 0, 1, 2),
-      CheckAnswer(0, 1, 2),
+      AddData(input, 0.to(2): _*),
+      CheckAnswer(0.to(2): _*),
       StopStream,
-      AddData(input, 3, 4, 5),
+      AddData(input, 3.to(5): _*),
       StartStream(),
-      CheckAnswer(0, 1, 2, 3, 4, 5))
+      CheckAnswer(0.to(5): _*))
```

Comment:
Nit: the changes seems unnecessary. Also in Spark we usually use style `(1 to 5): _*` instead.

Reply:
Thanks for the pointer. I started from `(0 to 5)` but spark Scala style guide mentions avoiding infix notation so a bit puzzled (I was not sure `to` is an operator). Will update.